ujt/transformers.py

```diff
@@ -191,88 +209,85 @@ def generate_before_after_composite_slis(
 
 
 def apply_highlighted_edge_class_to_elements(elements, user_journey_name):
-    # We may want to refactor the edge map creation for testability.
-    # No other function currently requires us to map edge source to edge element.
-    edges_map = defaultdict(list)
-    nodes_list = []
-
-    for element in elements:
-        if utils.is_node_element(element):
-            nodes_list.append(element)
-        else:
-            # the edge originates from a client, but we want to know
-            # which user journey it is associated with
-            if "user_journey_name" in element["data"].keys():
-                edges_map[element["data"]["user_journey_name"]].append(element)
-            else:
-                edges_map[element["data"]["source"]].append(element)
-
-    # notice that remove/apply highlighted class mutate the edges map
-    remove_highlighted_class_from_edges(edges_map)
-
-    if user_journey_name:
-        apply_highlighted_class_to_edges(edges_map, user_journey_name)
-
-    # Usually, we avoid multiple for clauses in a list comprehension,
-    # but this is the canonical way to flattern lists in Python.
-    flattened_edges = [edge for edge_list in edges_map.values() for edge in edge_list]
-
-    return flattened_edges + nodes_list
-
-
-def remove_highlighted_class_from_edges(edges_map):
```

Comment:
We don't need this anymore since cytoscape elements should always start with empty class property every time update_graph_elements is called